### PR TITLE
#738: Added Http Integration Tests: Create tests to support for GET, GETDEL, GETEX, GETSET commands

### LINKED ISSUE #738: Http Integration Tests: Create tests to support HTTP(5)
This issue involves creating the below test cases for HTTP as well:

**Source**: integration_tests/commands/async/

- flushdb_test.go
- get_test.go
- getdel_test.go
- getex_test.go
- getset_test.go

Steps:
1) Create the corresponding tests for these commands in integration_tests/commands/http.
2) Use set_test.go as a reference, and how to alter the input and output to match JSON structures from RESP.
3) Run the tests and fix issues with command parsing in HTTP.

If you need to do anything **custom** to support HTTP, let @pratikpandey21 @lucifercr07 know.

### PR DESCRIPTION
**Pull request by karandixit10**
FIxes #738 

### Description

This pull request adds the Http Integration Tests for GET, GETDEL, GETEX, GETSET commands.

### CODE REVIEW

**Comment by karandixit10** [comment]
Need to add test for FLUSHDB also

File: internal/server/utils/redisCmdAdapter.go [hunk 1]
```diff
@@ -100,6 +106,13 @@ func ParseHTTPRequest(r *http.Request) (*cmd.RedisCmd, error) {
 						delete(jsonBody, key)
 						continue
 					}
+					if key == Members {
```
Comment (pratikpandey21) [inline internal/server/utils/redisCmdAdapter.go:109]: Can you add some corresponding tests in redisCmdAdapter_test for Member/Offset and Members?

Comment (karandixit10) [reply internal/server/utils/redisCmdAdapter.go:109]: Yeah I'll add these

**Review by lucifercr07 — APPROVED** [review]

**Comment by karandixit10** [comment]
Hi @lucifercr07  I have to add test for FLUSHDB also shall I create a different PR for that?

**Comment by karandixit10** [comment]
I was working on JSON.STRAPPEND command will add flushdb test later was having some issue in that

**Pull request merged**